solr/core/src/java/org/apache/solr/cloud/ZkController.java

```diff
@@ -2375,21 +2375,32 @@ public void rejoinShardLeaderElection(SolrParams params) {
 
   public void checkOverseerDesignate() {
```

Comment:
I saw that this function get called in coreContainer.load()? do we want to check this bit early (after creating a zkController?)